plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -207,22 +205,23 @@ def sweep(self, arguments: List[str]) -> None:
         log.info("Best parameters: " + str(best_parameters))
 
     def sweep_over_batches(
-        self,
-        ax_client: AxClient,
-        overrides: Sequence[Sequence[str]],
-        batch_of_trials_to_launch: BatchOfTrialType,
+        self, ax_client: AxClient, batch_of_trials: BatchOfTrialType,
     ) -> None:
         assert self.launcher is not None
         assert self.job_idx is not None
-        rets = self.launcher.launch(
-            job_overrides=overrides, initial_job_idx=self.job_idx
-        )
-        self.job_idx += len(rets)
-        for idx in range(len(batch_of_trials_to_launch)):
-            val = rets[idx].return_value
-            ax_client.complete_trial(
-                trial_index=batch_of_trials_to_launch[idx].trial_index, raw_data=val
+
+        chunked_batches = self.chunks(batch_of_trials, self.max_batch_size)
+        for batch in chunked_batches:
+            overrides = [x.overrides for x in batch]
+            rets = self.launcher.launch(
+                job_overrides=overrides, initial_job_idx=self.job_idx
```

Comment:
are you not collecting the return values from the launcher?

Reply:
We do not carry around the return values. We just update the AxClient state.